.github/workflows/base_tests.yml

```diff
@@ -71,14 +88,15 @@ jobs:
       - name: Install p2pd
         run: |
           V=1 bash scripts/build_p2pd.sh p2pdCache 124530a3
+      
+      - name: Install deps
+        run: |
+          nimble install -y --depsOnly
 
       - name: Run tests
         run: |
           nim --version
           nimble --version
-          nimble install -y --depsOnly
```

Comment:
why has `nimble install -y --depsOnly` been deleted?

Reply:
Not deleted, moved. I'm trying to remove one of the (almost) duplicate files. Currently in the process of checking whether everything behaves the same, and tests pass.